js/chat/chat.ts

```diff
@@ -371,10 +395,12 @@ class ChatContainer extends LightElement {
       lastMessage.removeAttribute("is_streaming");
       this.#finalizeMessage();
       return;
+    } else {
+      lastMessage.setAttribute("is_streaming", "");
+      if (!message.chunk_type) {
+        lastMessage.setAttribute("content", message.content);
+      }
```

Comment:
I'm a little confused why this `if` is here. Because it is, the starting message will get dropped. I think for some providers that's OK, but IIRC some models like Gemini start with non-trivial content on the 1st iteration.